src/main/org/codehaus/groovy/ast/ClassCodeVisitorSupport.java

```diff
@@ -161,11 +161,6 @@ public void visitAssertStatement(AssertStatement statement) {
         super.visitAssertStatement(statement);
     }
 
-    public void visitBlockStatement(BlockStatement block) {
```

Comment:
Given the fact that it is still there on master, I assume it shouldn't be deleted. I would restore it and run the tests locally to make sure.